pytaxonkit.py

```diff
@@ -193,7 +193,7 @@ def list(ids, raw=False, threads=None, data_dir=None, debug=False):
     ...     subtaxa = [t for t in tree.traverse]
     ...     print(f'Top level result: {taxon.name} ({taxon.taxid}); {len(subtaxa)} related taxa')
     ...
-    Top level result: Solenopsis (13685); 305 related taxa
+    Top level result: Solenopsis (13685); 306 related taxa
```

Comment:
Just one of those clades for which they keep on finding/describing new taxa.

Reply:
Hmmm, I observed some different results for the above commands. I believe I'm on the right version and using the updated changes. Perhaps my Taxonomy database is a different version?

```
import pytaxonkit
>>> result = pytaxonkit.list([13685, 9903])
>>> for taxon, tree in result:
...     subtaxa = [t for t in tree.traverse]
...     print(f'Top level result: {taxon.name} ({taxon.taxid}); {len(subtaxa)} related taxa')
... 
Top level result: Solenopsis (13685); 304 related taxa
Top level result: Bos (9903); 29 related taxa
>>> pytaxonkit.__version__
'0.7.2+10.geaab060'
```